pkgs/top-level/all-packages.nix

```diff
@@ -4736,8 +4736,10 @@ with pkgs;
   eschalot = callPackage ../tools/security/eschalot { };
 
   espanso = callPackage ../applications/office/espanso {
-    inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation;
-    openssl = openssl_1_1;
+    inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation IOKit Kernel;
+  };
+  espanso-wayland = espanso.override {
+    waylandSupport = true;
   };
```

Comment:
```suggestion
```
we usually do not do variants for wayland support unless it is not possible ot compile it in by default

Reply:
As per [the espanso documentation](https://espanso.org/docs/install/linux/#wayland-compile), you can only compile with either X11 or Wayland support, not both in the same binary.